pure3.md

```diff
@@ -0,0 +1,310 @@
+than 100 companies and nine government agencies, the cause of the attack was
+Sunburst malware that had been uploaded via a routine update from Orion software
+produced by SolarWinds. The attack taught both public and private concerns some
+hard lessons, such as:
+
+* The very thing designed to protect the security of software-patches and
+  updates that fix routine vulnerabilities-can also be used to deliver malware
+  that can damage or corrupt.
+* As software products contain third-party components, program developers are
+  not in complete control of the contents of their products.
+* Malware can linger undetected for extended periods of time, allowing it to
+  infect every system that touches it. The Sunburst malicious code was part of
+  an update released by SolarWinds in the spring of 2020, yet it was not
+  reported until December of that year.
+
+Automotive software supply chains are particularly tempting targets for
+attackers. This is because automotive software runs on a large number of
+vehicles and updating all of them to recover quickly from an attack is a
+slow and difficult proposition. As such, a successful software supply chain
+attack means that thousands (or more) of vehicles are impacted. Any attack that
+targets a large number of vehicles on roads can do immeasurable damage. Further,
+while Uptane can securely deliver updates to vehicles to recover from the
+attack, ensuring every vehicle on the road receives the update immediately is
+very difficult given the circumstances of use of these vehicles. Therefore,
+strategies that can secure automotive software supply chains are vital to
+_avoid_ such attacks altogether.
+
+Governments around the world have recognized this fact. Regulations are
+forthcoming from the United Nations and other governing bodies, and standards
+have been published that specify how software running on automobiles must be
+secured. President Biden of the United States of America issued an Executive
+Order describing the need to improve the nation's cybersecurity. This EO, in
+particular, pushes for solutions like Software Bills of Materials (SBOMs) that
+list the components of software artifacts. This information is vital in ensuring
+developers are aware of the components in their supply chain so that they can
+fix vulnerabilities as and when they are discovered. For example, an SBOM would
+help developers react quickly to situations like 2021's log4j incident by
+informing them of the software products that require patches or updates.
+
+Unfortunately, SBOMs do not provide end-to-end software supply chain security.
+They do not guarantee of the integrity of the operations performed as part of
+the software supply chain and the artifacts they consume and produce. They do
+not help with verifying that each operation was performed by authorized actors,
+or indeed that all required operations were in fact performed. An SBOM would not
```

Comment:
I added a few words at the beginning of this section so its clear why an SBOM would not have worked. If this connection is wrong, then please replace with the actual reason. 
```suggestion
or indeed that all required operations were in fact performed. For these reasons, an SBOM would not
```